sample_tracking/app/nav.tsx

```diff
@@ -41,17 +41,21 @@ export default function Nav() {
             });
         }
 
-    })
-
+        document.addEventListener("mousedown", (event) => {
+            const popupContainer = document.getElementById("add-sample-popup");
+            const addSampleButton = document.getElementById('add-sample-button');
+            if (addSampleButton?.contains(event.target)) {
+                setShowAddSampleMenu(!showAddSampleMenu)
+                return;
+            }
+            if (popupContainer?.contains(event.target)) {
+                console.log("Clicked Inside");
+            } else {
```

Comment:
we'll want to think about how errors are made visible to the user.  not sure console is that obvious, but its a reasonable stopgap